src/resources/postcss/tickets-admin/settings/_order_modifiers.pcss

```diff
@@ -0,0 +1,38 @@
+.tec-settings-order_modifier {
+	width: 30%;
+
+	label {
+		font-weight: bold;
+	}
+
+	input[type="date"],
+	input[type="datetime-local"],
+	input[type="datetime"],
+	input[type="email"],
+	input[type="month"],
+	input[type="number"],
+	input[type="password"],
```

Comment:
is there a password field in there? Most of these? Why override them?

Reply:
@redscar can you help with that ?